tensorflow/src/main/scala/com/spotify/featran/tensorflow/package.scala

```diff
@@ -17,26 +17,32 @@
 
 package com.spotify.featran
 
+import org.tensorflow.example.{Example, Features}
 import org.tensorflow.{example => tf}
 
 package object tensorflow {
 
+  case class TensorFlowFeatureBuilder(
+    @transient private val underlying: Features.Builder = tf.Features.newBuilder())
+      extends FeatureBuilder[tf.Example] {
+    override def init(dimension: Int): Unit = underlying.clear()
+    override def add(name: String, value: Double): Unit = {
+      val feature = tf.Feature
+        .newBuilder()
+        .setFloatList(tf.FloatList.newBuilder().addValue(value.toFloat))
+        .build()
+      underlying.putFeature(name, feature)
+    }
+    override def skip(): Unit = Unit
+    override def skip(n: Int): Unit = Unit
+    override def result: tf.Example =
+      tf.Example.newBuilder().setFeatures(underlying).build()
+
+    override def newBuilder: FeatureBuilder[Example] = TensorFlowFeatureBuilder(underlying.clear())
```

Comment:
Are we reusing the `Features.Builder` in this case? That'll cause problem I think?

Reply:
@nevillelyh yeah good point